src/lib.rs

```diff
@@ -617,28 +623,25 @@ impl<N: Name> Type<N> {
     /// let t = tp!(@arrow[tp!(0), tp!(1)]);
     /// assert_eq!(format!("{}", &t), "t0 → t1");
     ///
-    /// let mut ctx = Context::default();
-    /// ctx.extend(0, tp!(int));
-    /// let fvs_computed = t.free_vars(&ctx);
-    /// let fvs_expected = vec![1];
+    /// let mut vs_computed = t.vars();
+    /// vs_computed.sort();
+    /// let vs_expected = vec![0, 1];
     ///
-    /// assert_eq!(fvs_computed, fvs_expected);
+    /// assert_eq!(vs_computed, vs_expected);
```

Comment:
we could just replace the `vs_expected` definition and but it directly into the assertion, for brevity.